.github/workflows/tests.yml

```diff
@@ -24,16 +32,14 @@ jobs:
 
   gen-test-matrix:
     name: Find test files
-    if: "!contains(github.event.head_commit.message, '[skip ci]')"
-    runs-on: [self-hosted]
+    runs-on: ubuntu-latest
```

Comment:
Ingenious

Reply:
> The PR also switches to using GitHub hosted test runners for all jobs other that the matrix job in which tests are run, on the basis that this is where the self-hosted runners provide most value, and running the other jobs on GitHub runners means they won't be competing with the long running tests.

On the other hand these jobs will have to compete with all other repositories in this organisation using the GitHub hosted runners.  I guess we can see how it goes, and revert it back in case there's too much traffic.

Reply:
Is there somewhere to see the queued jobs for the org?

Reply:
https://github.com/organizations/UCL/settings/actions/hosted-runners.  Unfortunately it doesn't look like there's an org-level view of past jobs, to get an idea of usual traffic.

Reply:
Yeah agree that would be good to keep an eye on this as there isn't a strong gain to running these on shared GitHub hosted runners rather than self-hosted.